src/components/modal/ModalSubmit.ts

```diff
@@ -0,0 +1,195 @@
+import { createHTMLElement } from '../../utils/createHTMLElement';
+import { createInputElement } from '../../utils/createInputElement';
+import { state } from "../../state/State";
+
+class ModalSubmit {
+  constructor() {
+    
+  }
+
+  private handlerInput = (e: Event): void => {
+    const input = e.target as HTMLInputElement;
+    const inputRequest: string = input.value;
+
+    if (input.classList.contains('name')) {
+      if (e.type === 'input') {
+        input.value = inputRequest.replace(/[^A-Za-zА-Яа-я-'\s]+/g, '');
+      }
+      if (e.type === 'change') {
+        input.value = input.value.trim();
+      }
+    }
+
+    if (input.classList.contains('number-phone')) {
+      if (e.type === 'click') {
+        if (inputRequest.length === 0) {
+          input.value = '+';
+        }
+      }
```

Comment:
These 2 conditions can be combined into one expression.
```suggestion
      if (e.type === 'click' && inputRequest.length === 0) {
        input.value = '+';
      }
```